Cython/Compiler/Main.py

```diff
@@ -156,7 +156,7 @@ def _split_qualified_name(self, qualified_name):
             is_package = False
             for suffix in ('.py', '.pyx'):
                 path = self.search_include_directories(
```

Comment:
This seems a classical case of "function does too much", but since that's not the fault of this PR, I'd leave it at that.